src/main/java/org/homeschoolpebt/app/cli/TransmitterCommands.java

```diff
@@ -54,17 +60,23 @@ public void transmit() throws IOException, JSchException, SftpException {
       submissionIds.add(i.getId());
     });
 
+    Map<String, List<String>> appIdToLaterDoc = new HashMap<>();
```

Comment:
The type looks wrong, since will only ever contain empty ArrayLists? Perhaps

`HashSet<String> laterDocAppIds`

?